src/tools/tests/goldens/kotlin-connection-type-generator.cgtest

```diff
@@ -1,60 +1,60 @@
-[header]
+-----[header]-------------------------------------------------------------------
```

Comment:
Just curious. Is this expected to be 80 characters? If the intention is to visually separte the sections better, something shorter like `----[header]----` or `#### [header] ####`might suffice?

Reply:
Visual separation was the intention. I can change to ----[header]---- if you prefer that.

Reply:
Done for you enjoyment :)

Reply:
Thanks! :)